app/src/main/java/com/example/coinage/fragments/AddTransactionFragment.java

```diff
@@ -16,30 +17,38 @@
 import android.widget.Button;
 import android.widget.DatePicker;
 import android.widget.EditText;
+import android.widget.Toast;
 
 import com.example.coinage.R;
+import com.example.coinage.models.Budget;
+import com.example.coinage.models.Spending;
 import com.example.coinage.models.Transaction;
+import com.parse.FindCallback;
+import com.parse.ParseQuery;
 import com.parse.ParseUser;
 import com.parse.SaveCallback;
 
 import java.math.BigDecimal;
 import java.text.ParseException;
 import java.util.Calendar;
 import java.util.Date;
+import java.util.List;
 import java.util.Locale;
 
 // track a new transaction by entering information (date, amount, category, description)
 public class AddTransactionFragment extends Fragment {
     public static final String TAG = "AddTransactionFragment";
 
     public final Calendar myCalendar = Calendar.getInstance();
-    public static final String myFormat="MM/dd/yy";
+    public static final String myFormat = "MM/dd/yy";
```

Comment:
nit - if you need to fix formatting issues, best to do it in a separate diff! Try to have one purpose per diff (you can bundle a bunch of formatting fixes into a single diff though to save time)